example/web/index.html

```diff
@@ -0,0 +1,146 @@
+            }
+          }
+        var o = function () { this._q = []; return this }
+          ; var a = ["add", "append", "clearAll", "prepend", "set", "setOnce", "unset", "preInsert", "postInsert", "remove"]
+          ; for (var c = 0; c < a.length; c++) { s(o, a[c]) } n.Identify = o; var u = function () {
+            this._q = []
+        ; return this
+          }
+          ; var l = ["setProductId", "setQuantity", "setPrice", "setRevenueType", "setEventProperties"]
+          ; for (var p = 0; p < l.length; p++) { s(u, l[p]) } n.Revenue = u
+          ; var d = ["init", "logEvent", "logRevenue", "setUserId", "setUserProperties", "setOptOut", "setVersionName", "setDomain", "setDeviceId", "enableTracking", "setGlobalUserProperties", "identify", "clearUserProperties", "setGroup", "logRevenueV2", "regenerateDeviceId", "groupIdentify", "onInit", "logEventWithTimestamp", "logEventWithGroups", "setSessionId", "resetSessionId"]
+          ; function v(e) {
+            function t(t) {
+        e[t] = function () {
+          e._q.push([t].concat(Array.prototype.slice.call(arguments, 0)))
+        }
+            }
+            for (var n = 0; n < d.length; n++) { t(d[n]) }
+          } v(n); n.getInstance = function (e) {
+            e = (!e || e.length === 0 ? "$default_instance" : e).toLowerCase()
+        ; if (!Object.prototype.hasOwnProperty.call(n._iq, e)) {
+          n._iq[e] = { _q: [] }; v(n._iq[e])
+        } return n._iq[e]
+          }; e.amplitude = n
+      })(window, document);
+</script>
+
+  <!-- This script installs service_worker.js to provide PWA functionality to
+       application. For more information, see:
+       https://developers.google.com/web/fundamentals/primers/service-workers -->
+  <script>
+    var serviceWorkerVersion = null;
+    var scriptLoaded = false;
+    function loadMainDartJs() {
+      if (scriptLoaded) {
+        return;
+      }
+      scriptLoaded = true;
+      var scriptTag = document.createElement('script');
+      scriptTag.src = 'main.dart.js';
+      scriptTag.type = 'application/javascript';
+      document.body.append(scriptTag);
+    }
+
+    if ('serviceWorker' in navigator) {
+      // Service workers are supported. Use them.
+      window.addEventListener('load', function () {
+        // Wait for registration to finish before dropping the <script> tag.
+        // Otherwise, the browser will load the script multiple times,
+        // potentially different versions.
+        var serviceWorkerUrl = 'flutter_service_worker.js?v=' + serviceWorkerVersion;
+        navigator.serviceWorker.register(serviceWorkerUrl)
+          .then((reg) => {
+            function waitForActivation(serviceWorker) {
+              serviceWorker.addEventListener('statechange', () => {
+                if (serviceWorker.state == 'activated') {
+                  console.log('Installed new service worker.');
+                  loadMainDartJs();
```

Comment:
Just curious does this work in service worker? I remember we have our customers asked about JS SDK support in service worker.